src/ast/ClassDecl.ts

```diff
@@ -70,4 +72,8 @@ export class ClassDecl extends Content {
         this.fields.forEach((fieldDecl: FieldDecl) => fieldDecl.typeCheck());
         this.functions.forEach((funcDecl: FuncDecl) => funcDecl.typeCheck());
     }
+
+    public getAbsolutePath(): string {
+        return this.parentPath + "/" + this.className + ".ts";
```

Comment:
Just as an aside, you can get away from using string concatenation by using template literals
```TypeScript
return `${this.parentPath}/${this.className}.ts`;
```

Reply:
Yeah there are some other areas where I use string interpolation instead of concatenation, i can change this for sure